src/TraceEvent/Microsoft.Diagnostics.Tracing.TraceEvent.nuspec

```diff
@@ -44,17 +46,17 @@
     <file src="Microsoft.Diagnostics.Tracing.TraceEvent.props" target="build\Microsoft.Diagnostics.Tracing.TraceEvent.props" />
 
     <!-- Native binaries -->
-    <file src="$OutDir$netstandard2.0\amd64\KernelTraceControl.dll" target="lib\native\amd64\KernelTraceControl.dll" />
-    <file src="$OutDir$netstandard2.0\amd64\msdia140.dll" target="lib\native\amd64\msdia140.dll" />
-    <file src="$OutDir$netstandard2.0\amd64\msvcp140.dll" target="lib\native\amd64\msvcp140.dll" />
-    <file src="$OutDir$netstandard2.0\amd64\vcruntime140.dll" target="lib\native\amd64\vcruntime140.dll" />
-    <file src="$OutDir$netstandard2.0\amd64\vcruntime140_1.dll" target="lib\native\amd64\vcruntime140_1.dll" />
-
-    <file src="$OutDir$netstandard2.0\x86\KernelTraceControl.dll" target="lib\native\x86\KernelTraceControl.dll" />
-    <file src="$OutDir$netstandard2.0\x86\KernelTraceControl.Win61.dll" target="lib\native\x86\KernelTraceControl.Win61.dll" />
-    <file src="$OutDir$netstandard2.0\x86\msdia140.dll" target="lib\native\x86\msdia140.dll" />
-    <file src="$OutDir$netstandard2.0\x86\msvcp140.dll" target="lib\native\x86\msvcp140.dll" />
-    <file src="$OutDir$netstandard2.0\x86\vcruntime140.dll" target="lib\native\x86\vcruntime140.dll" />
+    <file src="$OutDir$netstandard2.0\amd64\KernelTraceControl.dll" target="build\native\amd64\KernelTraceControl.dll" />
```

Comment:
I would not be surprised to find that this breaks people who are copying files from the NuGet package directory.  Does this fix anything other than the warning?  I'm inclined to just suppress or ignore this warning.

Reply:
The files are still in the package, but now they will be in the expected location in the package. The files are copied via the custom build logic which was updated to match.

Reply:
So I definitely see that the TraceEvent/PerfView build should be fine.  I'm just wondering if other folks might be copying files out without using TraceEvent.props.  I suspect that we should be fine here because when folks ingest the new package they'll experience this and have the opportunity to fix it.